charts/linkerd2/artifacthub-repo.yml

```diff
@@ -0,0 +1,14 @@
+#repositoryID: The ID of the Artifact Hub repository where the packages will be published to (optional, but it enables verified publisher)
```

Comment:
This comment makes it sound like this is needed for the verified publisher badge?

Reply:
Correct, but it's a two step process. After including this file in our Helm repo we'll be able to "claim" it in the artifacthub UI, which will give us this ID, to be added into this file and pushed again.